static/style.css

```diff
@@ -165,10 +167,14 @@ a.ButtonSquare:hover {
 
 #popup > div img {
 	width: 96px;
-	height: 96px}
+	height: 96px;
+	position: static;
+  	align-self: center;
```

Comment:
This two properties are useless because you already center the image with the class `#popup > div .ButtonRound`